apps/testclient/urls.py

```diff
@@ -3,10 +3,10 @@
                     test_userinfo, test_userinfo_v2,
                     test_metadata, test_metadata_v2,
                     test_openid_config, test_openid_config_v2,
-                    test_coverage, test_coverage_v2, test_patient, test_patient_v2, test_links)
+                    test_coverage, test_coverage_v2, test_patient, test_patient_v2, test_links, openapi)
 
 urlpatterns = [
-
+    url(r'^openapi$', openapi, name='bluebutton_openapi'),
```

Comment:
I don't think we want this to live under the testclient... maybe like `http://localhost:8000/docs/openapi` or like `http://localhost:8000/docs/api/openapi`.... currently its at `http://localhost:8000/testclient/openapi`. @dtisza1 what were your thoughts?

Reply:
can be move to a dedicated app